### DISCUSSION TITLE
Lock a color to not fill it by accident.

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6559] [Ideas]
### Is your feature request related to a problem? Please describe

When you use the "Lines and Areas" option of the fill tool to, for example, change the line and fill from one color to another in a specific area of ​​the drawing, if you move it too close to another color, that color will also show up affected by padding.

### Describe the solution you'd like

The idea would be able to lock that color so that it cannot be filled by accident.

**Comment by RodneyBaker** [comment]
I'm adding 'needs info' here to get as much information as possible as I think I know what is being request but not entirely sure.  Developers interested in taking on the request might have additional questions.

I suppose the main underlying feature I get from this request is the ability to lock down color styles.
Presumably that might be a little like treating the style as if that specific color added to the Alpha Channel and (for the purpose of color filling) the Lock Alpha option is on.  (Although, currently there is no such thing as Lock Alpha with Fill Tool)

**Comment by RodneyBaker** [comment]
This isn't quite #5989 but has some simularities.

Also, we can currently lock palettes but as far as I know we cannot lock individual styles.
It would be good to know if Opentoonz does in fact have that individual style locking capability.

**Comment by Abel032** [comment]
I'm not sure what is the exact situation.
But if you want to only change the paint and lines of one style(the line is bound of paint), it's already supported in area mode.(Change both the color and the line next to it with autoPaint feature on).
